docs/storage/storage.md

```diff
@@ -177,6 +177,14 @@ We strongly recommend using a hardware RAID system instead of software RAID.
 We do not provide support for issues resulting from the choice of software RAID for storage repositories.
 :::
 
+:::info
+An automated RAID health check is performed regularly using the `check_raid` plugin.
+
+This plugin is executed automatically every ten minutes by Xen Orchestra on each host. When a check is in progress, Xen Orchestra displays an alert to notify users that the RAID status is being verified:
```

Comment:
This doesn't really explain why it does so, and as a user one would be legitimate to ask "why display it at all?".